hail_search/search.py

```diff
@@ -3,13 +3,18 @@
 
 def search_hail_backend(request):
     sample_data = request.pop('sample_data', {})
+    genome_version = request.pop('genome_version')
 
     data_types = list(sample_data.keys())
     single_data_type = data_types[0] if len(data_types) == 1 else None
 
     sample_data = sample_data[single_data_type]
-    data_type = single_data_type
     query_cls = QUERY_CLASS_MAP[single_data_type]
 
-    query = query_cls(data_type, sample_data=sample_data, **request)
+    query = query_cls(sample_data, genome_version, **request)
     return query.search()
+
+
+def load_globals():
+    for cls in QUERY_CLASS_MAP.values():
+        cls.load_globals()
```

Comment:
I need clarification on why loading once per data type is sufficient.

Reply:
why would it not be? We only have one annotation table per genome version per data type, so we load the globals for each data type for each genome version, which means we have loaded all the possible globals

Reply:
I got it.